tests/test_runner.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import os
+import subprocess
+import unittest
+
+import django
+import pytest
+from django.test import SimpleTestCase
+
+
+@pytest.mark.skip(reason="Run these unittest tests in a sub-process.")
+class UnitTestRunnerTests(SimpleTestCase):
+    def test_traceback(self):
+        raise ValueError
+
+    def test_assertion(self):
+        self.assertEqual(1, 2)
+
+    def test_django_assertion(self):
+        "Tests can have descriptions."
+        self.assertURLEqual("/url/", "/test/")
+
+    def test_pass(self):
+        self.assertEqual(1, 1)
+
+    @unittest.skip("A skipped test.")
+    def test_skip(self):
+        raise ValueError
+
+    @unittest.expectedFailure
+    def test_expected_fail(self):
+        self.assertEqual(1, 2)
+
+    @unittest.expectedFailure
+    def test_unexpected_success(self):
+        self.assertEqual(1, 1)
+
+
+class TestRunnerTests:
+    env = {
+        **os.environ,
+        "DJANGO_SETTINGS_MODULE": "tests.settings",
+    }
+
+    def test_traceback(self):
+        stderr = subprocess.run(
```

Comment:
I think we can make a helper function. Also we should check for return code and stdout

Reply:
Done!